File: Website/InfoTerraTerra/InfoTerraTerra_Library/Tracking/SingleVolantinoStatistics.cs
namespace InfoTerraTerra_Library.Tracking;

public class SingleVolantinoStatistics
{
    public required string TitoloVolantino { get; init; }
    public required KeyValuePair<string, string>[] Counters { get; init; }
    public required KeyValuePair<string, string>[] PerCitta { get; init; }
    public required KeyValuePair<string, string>[] PerVia { get; init; }
    public required KeyValuePair<string, string>[] PerLuogo { get; init; }
}

File: dataLayer/tracking/SingleVolantinoStatistics.ts
import {KeyValuePair} from "../../model/KeyValuePair";

export interface SingleVolantinoStatistics {
    titoloVolantino: string;
    counters: KeyValuePair<string, string>[];
    perCitta: KeyValuePair<string, string>[];
    perVia: KeyValuePair<string, string>[];
    perLuogo: KeyValuePair<string, string>[];
}